bitrise.yml

```diff
@@ -4,32 +4,36 @@ default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 workflows:
   check:
     steps:
-      - git::https://github.com/bitrise-steplib/steps-check.git: { }
+    - git::https://github.com/bitrise-steplib/steps-check.git:
 
   e2e:
     steps:
-      - git::https://github.com/bitrise-steplib/steps-check.git:
-          inputs:
-            - workflow: e2e
+    - git::https://github.com/bitrise-steplib/steps-check.git:
+        inputs:
+        - workflow: e2e
 
   sample:
     envs:
-      - TEST_APP_URL: https://github.com/bitrise-samples/sample-apps-ios-cocoapods-no-shared-schemes.git
-      - TEST_APP_BRANCH: master
-      - BITRISE_PROJECT_PATH: SampleAppWithCocoapods/SampleAppWithCocoapods.xcworkspace
+    - TEST_APP_URL: https://github.com/bitrise-samples/sample-apps-ios-cocoapods-no-shared-schemes.git
+    - TEST_APP_BRANCH: master
+    - BITRISE_PROJECT_PATH: SampleAppWithCocoapods/SampleAppWithCocoapods.xcworkspace
     steps:
-      - script:
-          inputs:
-            - content: |-
-                #!/bin/bash
-                set -ev
-                rm -rf _tmp
-      - git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone.git@master:
-          inputs:
-            - repository_url: $TEST_APP_URL
-            - branch: $TEST_APP_BRANCH
-            - clone_into_dir: ./_tmp
-      - path::./:
-          title: Step Test
-          inputs:
-            - project_path: ./_tmp/$BITRISE_PROJECT_PATH
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ev
+            rm -rf _tmp
+    - git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone.git@master:
+        inputs:
+        - repository_url: $TEST_APP_URL
+        - branch: $TEST_APP_BRANCH
+        - clone_into_dir: ./_tmp
+    - path::./:
+        title: Step Test
+        inputs:
+        - project_path: ./_tmp/$BITRISE_PROJECT_PATH
+
+  generate_readme:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-readme-generator.git@main:
```

Comment:
Do we have to write `main` if that is the "default" branch?

Reply:
Sadly, yes: https://bitfall.slack.com/archives/CT108S18S/p1629716232037200?thread_ts=1629715585.035900&cid=CT108S18S.

Reply:
Sadly yes. I'll send you the link of the internal discussion on Slack.

**tl;dr:** Bitrise only supports `master` as the default branch during `bitrise.yml` parse at the moment.